src/Home/types.js

```diff
@@ -11,10 +12,6 @@ const Types = () => {
     setTypes(response.data.results);
```

Comment:
pokeapi gives us `"unknown"` and `"shadow"` types, neither of which are really relevant for our purposes. i would say leave it if not for the fact that these two types result in uncolored pills at the end of the home screen type list. lowkey i'm down to manually prune them.